lib/python/rose/config_editor/valuewidget/array/logical.py

```diff
@@ -179,8 +184,25 @@ def populate_table(self):
             self.del_button.hide()
         else:
             self.del_button.show()
+        if self.has_titles:
+            for c, label in enumerate(self.metadata['element-titles']):
+                if c >= len(table_widgets):
```

Comment:
There's an inconsistency with how/where things are bunged into the front end between the various array implementations so not all of them need the offset.